### PR TITLE
Fix after #5203

### PR DESCRIPTION
**Pull request by robert00s**
https://github.com/koreader/koreader/pull/5203#issuecomment-522156268
https://github.com/koreader/koreader/pull/5203#pullrequestreview-276228925

### CODE REVIEW

File: frontend/apps/reader/modules/readerfooter.lua [hunk 1]
```diff
@@ -565,7 +565,7 @@ function ReaderFooter:addToMainMenu(menu_items)
             {
                 text = _("Alignment"),
                 enabled_func = function()
-                    return self.settings.disable_progress_bar
+                    return not self.settings.disable_progress_bar == false
                 end,
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerfooter.lua]: Maybe some parentheses for clarity?

**Comment by poire-z** [comment]
Just in case you're not yet bored with the footer :)
As you show some sample in the menu item text for Progress percentage format, you could show it for more/all of them (most probably just useful for the Duration format, classic vs modern is a bit obscure -  although one immediately sees what it means in the footer when updating...)

File: frontend/apps/reader/modules/readerfooter.lua (hunk 1 above)
Comment (poire-z) [reply frontend/apps/reader/modules/readerfooter.lua]: Or just `return self.settings.disable_progress_bar == true`.
(our touchmenu code considers `nil` to mean: enabled - it just checks for `false` meaning: disabled)

**Comment by robert00s** [comment]
Now all settings have samples:

obraz

obraz

obraz

obraz

**Pull request merged**